File: python/examples/example_masterdata.py
from vesselfinder_api import VesselFinderApi
from vesselfinder_api.exceptions import ApiErrorException

v = VesselFinderApi(userkey='-- Input your userkey here --', errormode=False, save_last_info=True)

try:
    print(v.master_data(imo=[9466233, 9723708, 9328649]))
except ApiErrorException as e:
    print(e)

File: php/examples/example_masterdata.php
<?php
include __DIR__ . '/../VesselFinderApi.php';

use API\VesselFinderApi;

$client = new VesselFinderApi('-- Input your userkey here --');

try {
    $masterdata = $client->masterData([9466233, 9723708, 9328649]);
    print_r($masterdata);
} catch (\Exceptions\ApiErrorException $e) {
    print_r($e->getMessage() . PHP_EOL);
}